### PR TITLE
add option to map to limit zoom out too much

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by varffs** [comment]
Should we also have a max zoom value (assuming that also exists)?

**Comment by varffs** [comment]
I can't seem to check this as some CSS rules are hiding the map and I don't know why

**Comment by mixcas** [comment]
I don't feel a max zoom limit is needed. Most ppl know how to google maps, no? Could be annoying if markers are too close. I've actually experience this annoyance on the [facebook.com/live](http://facebook.com/live) map

plz debug that issue on your side, is not happening to me. maybe some plugin is blocking it or never actully loading.

**Comment by mikeyrayvon** [comment]
an initial max zoom would be good

File: src/js/main.js [hunk 1]
```diff
@@ -589,11 +589,28 @@ Site.Map = {
...
+
+      // Listen for bounds change (only once)
+      google.maps.event.addListenerOnce(_this.map, 'bounds_changed', function(event) {
+        //remove one zoom level to ensure no marker is on the edge.
+        _this.map.setZoom(_this.map.getZoom()-1);
+
+        // set a minimum zoom
+        // if you got only 1 marker or all markers are too close to each other map will be zoomed too much.
+        if (_this.map.getZoom() > 13) {
+          _this.map.setZoom(4);
```
Comment (varffs) [inline src/js/main.js:623]: why does this revert to 4 rather than 13?

Comment (mixcas) [reply src/js/main.js:623]: cuz 4 is nice zoom level, otherwise reverting to 13 still too close

Comment (varffs) [reply src/js/main.js:623]: ok

**Pull request merged**